gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java

```diff
@@ -170,12 +173,28 @@ public GrpcCallContext withStreamIdleTimeout(@Nullable Duration streamIdleTimeou
     }
 
     return new GrpcCallContext(
-        channel, callOptions, streamWaitTimeout, streamIdleTimeout, channelAffinity);
+        channel, callOptions, streamWaitTimeout, streamIdleTimeout, channelAffinity, extraHeaders);
   }
 
+  @BetaApi("The surface for channel affinity is not stable yet and may change in the future.")
```

Comment:
I wanted to note that you can't add `@BetaApi` to stable features that have been pushed in a release already. Fortunately for you, the whole class is already marked `@BetaApi`, so this is ok.

Reply:
This was my fault. I should have added it in my previous PR.